soknad/src/main/java/no/nav/k9/søknad/ytelse/olp/v1/kurs/Kurs.java

```diff
@@ -31,13 +32,18 @@ public class Kurs {
     @Valid
     private List<KursPeriodeMedReisetid> kursperioder;
 
+    @JsonProperty(value = "institusjonUuid")
+    @Valid
+    private UUID institusjonUuid;
```

Comment:
Bør legge til validering på at det enten settes istitusjonUuid eller holder, men ikke begge to eller ingen av dem